test/test-signal-pending-on-close.c

```diff
@@ -91,4 +96,24 @@ TEST_IMPL(signal_pending_on_close) {
   return 0;
 }
 
-#endif
\ No newline at end of file
+
+TEST_IMPL(signal_close_loop_alive) {
+  ASSERT(0 == uv_loop_init(&loop));
+  ASSERT(0 == uv_signal_init(&loop, &signal_hdl));
+  ASSERT(0 == uv_signal_start(&signal_hdl, stop_loop_cb, SIGPIPE));
+  uv_unref((uv_handle_t*) &signal_hdl);
+
+  ASSERT(0 == uv_kill(0, SIGPIPE));
```

Comment:
Sending the signal to the process group is causing same other test to fail in my setup. (I'm using autotools)
```
not ok 336 - tty
# exit code 6
# Output from process `tty`:
# width=383 height=102
# Assertion failed in test/test-tty.c on line 128: r == 0
```


Changing to the current process pid fixes the issue for me: `ASSERT(0 == uv_kill(uv_os_getpid(), SIGPIPE));`

Reply:
Good catch, thanks! Updated and I'll start a new CI.